Use install instead of cp in Makefile

This patch also changes the install target to copy the uncompressed man
page, so that dh_compress can compress it according to policy.

diff --git a/Makefile b/Makefile
index 47467e3..4b4fef2 100644
--- a/Makefile
+++ b/Makefile
@@ -42,7 +42,7 @@ clean:
 	rm -f $(OBJ) $(OBJ_ARCH) makedumpfile makedumpfile.8.gz
 
 install:
-	cp makedumpfile ${DESTDIR}/bin
-	cp makedumpfile-R.pl ${DESTDIR}/bin
-	cp makedumpfile.8.gz ${DESTDIR}/usr/share/man/man8
+	install -D -m755 ./makedumpfile ${DESTDIR}/usr/bin/makedumpfile
+	install -D -m755 ./makedumpfile-R.pl ${DESTDIR}/usr/bin/makedumpfile-R.pl
+	install -D -m755 ./makedumpfile.8 ${DESTDIR}/usr/share/man/man8/makedumpfile.8
 
